ipaserver/setup.py

```diff
@@ -76,4 +70,12 @@
                 'IPASecStore = ipaserver.secrets.store:IPASecStore',
             ],
         },
+        extras_require={
+            # These packages are currently not available on PyPI.
+            "caacl": ["pyhbac"],
```

Comment:
The missing `pyhbac` does not actually affect `caacl`, but `cert` - `cert-request` is the only place where the `acl_evaluate` function is called.